screens/Settings/SettingScreen.tsx

```diff
@@ -162,7 +161,7 @@ export const SettingScreen: React.FC<
 
             <AboutInji appId={controller.appId} />
 
-            {isAndroid() && <DataBackupAndRestore />}
+            <DataBackupAndRestore />
```

Comment:
No check required for the backup_restore_flag anymore?

Reply:
Yes, we dont have toggle from now